rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientRegisterActivity.java

```diff
@@ -49,4 +70,55 @@ protected Class getLoginPage() {
     protected PatientRegisterActivityPresenter createPatientRegisterActivityPresenter() {
         return new CovidPatientRegisterActivityPresenter(this);
     }
+
+    private Map<String, Locale> locales = new HashMap<>();
+    private String[] languages;
```

Comment:
Not clear to me why we need to keep a map and an array, can we just have the array if that's what the `AlertDialog` requires?

Reply:
We can always build the Locale from the language code saved in `SharedPreferences` if need be.

Reply:
done. now i have used `string-array`